README.md

```diff
@@ -66,27 +66,14 @@ This sections provides details on what you need install on your system in order
     - MacOS 10.12-10.13 (Sierra/High Sierra):
 
         ```
-        brew install node@6
+        brew install node@6.12.3
         ```
 
-- _(Recommended)_ Install nvm -- Used for Node.js version management
+- _(Recommended)_ Install nvm(<https://github.com/creationix/nvm#install-script>) -- Used for Node.js version management
```

Comment:
Maybe this makes more sense under the previous bullet point so people don't install Node.js system-wide and then see that we recommend not doing that?

```markdown
_(Recommended)_ Install using a version manager such as nvm.
    1. Install nvm following their instructions (https://github.com/creationix/nvm#installation)
    1. Install the correct version of Node.js using nvm:
    ```
    nvm install 6.12.3
    ```

Reply:
yeah I think keeping the nvm install under bullet point would enforce them install nvm and then switch to specific node version.